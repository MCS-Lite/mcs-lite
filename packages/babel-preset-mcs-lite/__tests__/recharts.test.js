import * as babel from 'babel-core';
import mcsLitePreset from '../index';

const babelOptions = {
  presets: [mcsLitePreset],
  babelrc: false,
};

const compile = code => babel.transform(code, babelOptions).code;

it('should work with recharts', () => {
  expect(
    compile(
      `
import { AreaChart } from 'recharts';
console.log(AreaChart);
  `
    )
  ).toMatchSnapshot();
});
