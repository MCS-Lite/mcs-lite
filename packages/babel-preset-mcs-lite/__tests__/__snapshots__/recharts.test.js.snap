// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should work with recharts 1`] = `
"'use strict';

require('recharts/lib/polyfill.js');

var _AreaChart = require('recharts/lib/chart/AreaChart.js');

var _AreaChart2 = _interopRequireDefault(_AreaChart);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

console.log(_AreaChart2.default);"
`;
